test/test_CppHeaderParser.py

```diff
@@ -1162,14 +1162,9 @@ def test_method_params(self):
             [],
         )
 
-    def test_class_template(self):
-        self.assertEqual(
```

Comment:
Hm? I don't really want to break existing users without a good reason. Why are existing tests changing?

Reply:
I think it's a side effect of the duplicated class `CarrotClass_TestCase`. When the class was duplicated, function `test_class_template` throws an exception but it was ignored by Python. When I renamed the second class to `ExternClass_TestCase`, Python catchs the exception and test fails.

If you look at class `CarrotClass` in header file you can see that the class is not template.